docs/v1.0/config/interpolation.md

```diff
@@ -46,7 +46,7 @@ monitors:
       hostname: db.local
 ```
 
-The result is `monitors.hostname = "db.local"` because `%{monitor.tags.hostname}` refers to the local `monitrs.tags.hostname` variable.
+The result is `monitors.hostname = "db.local"` because `%{monitor.tags.hostname}` refers to the local `monitors.tags.hostname` variable.
```

Comment:
Just wanted to check this is correct - monitors.tags.X vs monitor.tags.hostname?

Reply:
Yes, correct enough because haven't settled on a convention yet for this ("monitor" or "monitors": probably the latter since it's `monitors:` in the config file).